### PR TITLE
Add prettier configuration to format code automatically

### PR DESCRIPTION
**Pull request by bandax**

### CODE REVIEW

File: src/components/AddMovieWindow/AddMovieWindow.tsx [hunk 1]
```diff
@@ -1,165 +1,208 @@
...
+
+  if (!props.showModal) {
+    return null;
+  }
+
+  return (
+    <div className="add-movie-window">
+      <div className="add-movie-modal">
+        <h2 className="add-movie-title">Add movie</h2>
+        <a
```
Comment (endiny) [inline src/components/AddMovieWindow/AddMovieWindow.tsx:118]: make it a button

File: src/components/App.scss [hunk 2]
```diff
@@ -1,45 +1,68 @@
...
+}
+.col-10 {
+    width: 83.33%;
+}
+.col-11 {
+    width: 91.66%;
+}
+.col-12 {
+    width: 100%;
+}
```
Comment (endiny) [inline src/components/App.scss:40]: optional: You can use SCSS's loops for doing that

**Review by endiny — CHANGES_REQUESTED** [review]

File: src/components/App.scss (hunk 2 above)
Comment (bandax) [reply src/components/App.scss]: done

File: src/components/AddMovieWindow/AddMovieWindow.tsx (hunk 1 above)
Comment (bandax) [reply src/components/AddMovieWindow/AddMovieWindow.tsx]: done

**Pull request merged**